bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableClusterAdminGrpcClient.java

```diff
@@ -101,7 +101,6 @@ public void deleteCluster(DeleteClusterRequest request) {
   
   @Override
   public void close() throws Exception {
-    channel.close();
```

Comment:
Add a comment that things get closed behind the scenes. Does it need to be autocloseable now?